programs/bpf_loader/src/lib.rs

```diff
@@ -1544,13 +1533,12 @@ fn execute<'a, 'b: 'a>(
             Ok(info) => info,
             Err(e) => {
                 ic_logger_msg!(log_collector, "Failed to create SBF VM: {}", e);
-                return Err(InstructionError::ProgramEnvironmentSetupFailure);
+                return Err(Box::new(InstructionError::ProgramEnvironmentSetupFailure));
             }
         };
         create_vm_time.stop();
 
         execute_time = Measure::start("execute");
-        stable_log::program_invoke(&log_collector, &program_id, stack_height);
```

Comment:
Why did you remove this log and the one when the program fails? The invoke
context will log with the loader id right? So this add a log with the actual
program id which is useful

Reply:
`InvokeContext::process_executable_chain()` logs `program_id` as well, not `builtin_id`.

Reply:
Ah that's what the cursed confusing if was for lol 😂